tests/utils/test_dict_ops.py

```diff
@@ -54,7 +84,8 @@ def test_update_nested_pair(
         d2: dict[str, JsonType],
         expected: dict[str, JsonType],
     ):
-        assert update_nested_pair(d1, d2) == expected
+        pair = update_nested_pair(d1, d2)
+        assert pair == expected
```

Comment:
revert? the output is not a pair

Reply:
Sry left over from changing the tests 😄 reverted